en/components/date_range_picker.md

```diff
@@ -0,0 +1,431 @@
+
+```html
+<igx-date-range-picker>
+    <igx-picker-toggle igxSuffix>
+        <igx-icon>calendar_view_day</igx-icon>
+    </igx-picker-toggle>
+</igx-date-range-picker>
+```
+
+When Date Range Picker has two separate inputs, for start and end dates, it doesn't expose an calendar toggle by default. The [`IgxPickerToggleComponent`]({environment:angularApiUrl}/classes/igxpickertogglecomponent.html) should be manually added as a child for [`IgxDateRangeStartComponent`]({environment:angularApiUrl}/classes/igxdaterangestartcomponent.html) or [`IgxDateRangeEndComponent`]({environment:angularApiUrl}/classes/igxdaterangeendcomponent.html), as shown below:
+
+```html
+<igx-date-range-picker>
+    <igx-date-range-start>
+        ...
+        <igx-picker-toggle igxPrefix>
+            <igx-icon>calendar_view_day</igx-icon>
+        </igx-picker-toggle>
+        ...
+    </igx-date-range-start>
+    <igx-date-range-end>
+        ...
+    </igx-date-range-end>
+</igx-date-range-picker>
+```
+
+#### Dropdown mode
+
+The [`IgxDateRangePickerComponent`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html) component has two modes - dialog (default) and dropdown. To define the picker in dropdown mode, use the snippet below:
+
+```html
+<igx-date-range mode="dropdown"></igx-date-range>
+```
+
+<div class="sample-container loading" style="height:500px">
+    <iframe id="daterangepicker-mode-iframe" src="{environment:demosBaseUrl}/scheduling/daterangepicker-mode" width="100%" height="100%" seamless="" frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="daterangepicker-mode-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+When in dropdown mode, the `Done` button is not available. In both modes value is set when dates are picked from the calendar.
+
+#### Keyboard Navigation
+> Note: Use the demos for [Drop down mode](#dropdown-mode) and [Separate Editable Inputs](#separate-editable-inputs) to try the keyboard combinations defined in that topic section.
+
+To open/close [`IgxDateRangePickerComponent`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html) calendar, using keyboard, the [`mode`]([`IgxDateRangePickerComponent`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#mode)) should be `dropdown`.
+
+- <kbd>Alt</kbd> + <kbd>Down Arrow</kbd> - Opens the drop-down calendar and focuses it
```

Comment:
I would go with `dropdown` everywhere. Or `drop-down` or `drop down` - just to be same everywhere